sklearn/pipeline.py

```diff
@@ -803,10 +813,5 @@ def make_union(*transformers):
                               n_components=2, n_iter=5,
                               random_state=None, tol=0.0))],
            transformer_weights=None)
-
-
-    Returns
-    -------
-    f : FeatureUnion
     """
-    return FeatureUnion(_name_estimators(transformers))
+    return FeatureUnion(_name_estimators(transformers), **kwargs)
```

Comment:
Perhaps we should check, until we decide, perhaps, to implement some kind of `transformer_weights`, that only `n_jobs` is passed in. Sometimes it's better to be conservative. WDYT?